### PR TITLE
Move the thoth-station test apps to rick

### PR DESCRIPTION
**Pull request by harshad16**
Move the thoth-station test apps to rick
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

Depends-on: https://github.com/operate-first/apps/pull/1008
Related-to: https://github.com/thoth-station/thoth-application/issues/1769

### CODE REVIEW

**Comment by Gregory-Pereira** [comment]
If it's just migrating the namespace then it looks good. I'll let someone else approve to push it through though.

**Review by 4n4nd — COMMENTED** [review]
/lgtm

**Comment by HumairAK** [comment]
can you also update: https://github.com/operate-first/argocd-apps/blob/main/projects/thoth.yaml#L8 and add rick

**Review by tumido — CHANGES_REQUESTED** [review]
Marking @HumairAK's comment as "request changes" https://github.com/operate-first/argocd-apps/pull/204#issuecomment-921224874

Please add the Rick cluster to your project:
https://github.com/operate-first/argocd-apps/blob/b3165a430accd2655131957889f96c13286972fe/projects/thoth.yaml#L8-L10

**Comment by goern** [comment]
@tumido @HumairAK see https://github.com/operate-first/argocd-apps/pull/209

**Comment by harshad16** [comment]
@HumairAK @tumido The required changes are already completed.
Ready for review again

File: envs/emea/rick/thoth/test/test-thoth-investigator.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ spec:
     path: investigator/overlays/zero-test
```
Comment (HumairAK) [inline envs/emea/rick/thoth/test/test-thoth-investigator.yaml:10]: shouldn't this be `rick-test` now?

Comment (harshad16) [reply envs/emea/rick/thoth/test/test-thoth-investigator.yaml]: Thanks , it is resolved.
PTAL @HumairAK

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

**Pull request merged**